### DISCUSSION TITLE
Publish fdroid variant apk on GitHub releases

### DISCUSSION THREAD

**Discussion by alexwaibel** [element-hq/element-meta#2248] [Ideas]
I noticed the version of element I installed from the releases section in GitHub is the Google play variant which requires play services to deliver notifications.

I would like to be able to download and install the version which does not rely on play services directly, without having to go through fdroid. This would let me use something like [obtainium](https://github.com/ImranR98/Obtainium) to install the app and keep it up to date without relying on fdroid to build and sign it.

**Comment by einarpersson** [comment]
I am also using fantastic [obtainium](https://github.com/ImranR98/Obtainium) (recommend to check it out) and came here for the same thing. 

I believe there is overlap among people looking for alternative social solutions (matrix) and alternative ways of managing apps on their phones (obtainium).
